src/GraphQL.Tests/Validation/ArgumentsOfCorrectTypeTests.cs

```diff
@@ -543,11 +542,7 @@ public void list_value_incorrect_item_type()
             ShouldFailRule(_ =>
             {
                 _.Query = query;
-                Rule.badValue(_, "stringListArg", "[String]", "[\"one\", 2]", 3, 36,
-                    new[]
-                    {
-                        "In element #1: Expected type \"String\", found 2."
-                    });
+                Rule.badValue(_, "stringListArg", "[String]", "[\"one\", 2]", 3, 36, "In element #2: [Expected type 'String', found 2.]");
```

Comment:
Notice change in index. Was 1, became 2. This is due to bug in `IsValidLiteralValue` before:
```csharp
 return listValue.Values
                        .SelectMany(value =>
                            IsValidLiteralValue(ofType, value, schema)
                                .Select((err, index) => $"In element #{index + 1}: {err}")
                        )
                        .ToArray();
```

Index should be taken over "external" list, not over internal `string[]` array of errors.